rtc_data_service/src/app_config.rs

```diff
@@ -42,3 +54,8 @@ impl AppConfig {
         conf.try_into()
     }
 }
+
+enum ValidationResult {
+    Invalid(String),
+    Valid,
+}
```

Comment:
Is this actually being used?

Reply:
Nope, removed: 24c2b407054703591ffbcd4fc2e49f4454f596e8